docs/how-to/kernel_language_cpp_support.rst

```diff
@@ -0,0 +1,210 @@
+.. meta::
+  :description: This chapter describes HIP's kernel language's C++ support.
+  :keywords: AMD, ROCm, HIP, C++ support
+
+################################################################################
+Kernel language C++ support
+################################################################################
+
+The HIP host API can be compiled with any conforming C++ compiler, as long as no
+kernel launch is present in the code.
+
+To compile device code and include kernel launches, a compiler with full HIP
+support is needed. On AMD platforms this is ``amdclang++``, whereas on NVIDIA
```

Comment:
I would remove the reference to NVIDIA and nvcc

Reply:
```suggestion
support is needed, such as ``amdclang++`` or ``hipcc``. For more information, see :doc:`ROCm compilers <llvm-project:index>`. 
```

Reply:
Removed the hipcc reference, but kept the link to ROCm compilers